docs/deployment/config_descriptions.md

```diff
@@ -18,6 +18,22 @@ dynamos:
     read: 5
 ```
 
+### config.yml Requirements
```

Comment:
We already have sample config.yml in the main deployment doc, which is already very long. I'd like to avoid making it longer, but if you think it's important I will make that change.

Reply:
Nah, I think it's fine as is